components/efuse/src/esp_efuse_utility.c

```diff
@@ -421,58 +421,74 @@ void esp_efuse_init_virtual_mode_in_flash(uint32_t offset, uint32_t size)
 void esp_efuse_utility_erase_efuses_in_flash(void)
 {
     if (esp_efuse_flash_offset == 0) {
-        ESP_EARLY_LOGE(TAG, "no efuse partition in partition_table? (Flash is not updated)");
+        ESP_EARLY_LOGE(TAG, "Virtual eFuse: no efuse partition in partition_table? (Flash is not updated)");
```

Comment:
I wonder if it would help if we change the `TAG` itself for the virtual EFuse configuration case?

Reply:
ya I agree that sounds better i'll update the PR

Reply:
updated.